syntax = "proto3";

package ai.wombo.tao.neuron;

import "tensor/protos/inputs.proto";

message MinerGenerationResponse {
  bytes frames = 1;
}

service Miner {
  rpc Generate(ai.wombo.tao.inputs.GenerationRequestInputs) returns (MinerGenerationResponse);
}
